frontends/web/src/routes/account/summary/filters.tsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2022 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { FunctionComponent } from 'react';
+import { useTranslation } from 'react-i18next';
+import { TChartFilters } from './types';
+import styles from './chart.module.css';
+
+
+type FilterProps = {
+  chartFiltersData: TChartFilters
```

Comment:
the displayWeek, displayMonth, displayYear, displayAll, callbacks are not really part of the data, just type them here?

also maybe rename to onDisplayWeek so it is clear that it's a callback fn?

Reply:
Unsure  what you meant by “type them here”? If you meant defining these fns inside `<Filter/>`, then it would be quite a hassle as the fns involved its parent's states, the chart settings/methods, etc so that means we have to pass all of these down to the `<Filter/>`

Imo, this becomes  (unnecessarily)  more complex.  I suggest that we should keep child components as “simple” or as “dumb” as possible, meaning we should try to keep these "complex" logic (functions) within the parent components,  and away from its children. This way it’s easier to test, refactor and re-use these child components.  

But I do agree with the naming! Was meaning to change that but totally forgot about it. I’ll also change the name to `chartFiltersProps` instead of `chartFiltersData`, as it’s true - these callback fns are not really “data”.